src/lib/api/api.go

```diff
@@ -119,7 +124,7 @@ func (a *API) Home(w http.ResponseWriter, r *http.Request) {
 	msg := "ping from " + r.RemoteAddr + ", sent pong in monitor"
 	log.Infoln(msg)
 	a.Data = msg
-	a.send(w, a.marshalled())
+	a.send(w, a.Data, "nil")
```

Comment:
@Rits1272 that's not exactly what I meant.
- In Go, nil is the zero value for pointers, interfaces, maps, slices, channels and function types, representing an uninitialized value.
- nil doesn't mean some "undefined" state, it's a proper value in itself. An object in Go is nil simply if and only if it's value is nil, which it can only be if it's of one of the aforementioned types.
- An error is an interface, so nil is a valid value for one, unlike for a string. For obvious reasons a nil error represents no error.

Reply:
Thank you for the review! I have changed the `message` from `string` to `interface`. So that we can now assign `nil` to the success message. Kindly review @aquibbaig :)